### PR TITLE
Add coturn servers table

### PR DESCRIPTION
**Pull request by Brutus5000**

### CODE REVIEW

**Comment by Askaholic** [comment]
Is the lobby server going to need to implement this? Or will it be handled by the API?

**Comment by Brutus5000** [comment]
API exposing the list to all clients

File: migrations/V124__coturn_servers.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+create table coturn_servers
+(
+    id            int(11) unsigned     not null auto_increment,
+    region        varchar(20)          not null comment 'Region of the server for simple user-based selection if latency checks don''t work',
+    host          varchar(255)         not null,
+    port          mediumint            not null,
```
Comment (Sheikah45) [inline migrations/V124__coturn_servers.sql]: The current coturn server have only hosts and not ports so this should be nullable

**Review by Sheikah45 — CHANGES_REQUESTED** [review]

File: migrations/V124__coturn_servers.sql (hunk 1 above)
Comment (Sheikah45) [reply migrations/V124__coturn_servers.sql]: Maybe the port should just be included in the host anyway?

Comment (Sheikah45) [reply migrations/V124__coturn_servers.sql]: Also shouldn't the unique be host and port?

**Review by Sheikah45 — APPROVED** [review]

**Pull request merged**